src/app/containers/visualizer/components/swt-surface-plot/swt-surface-plot.component.ts

```diff
@@ -346,13 +348,15 @@ export class SwtSurfacePlotComponent implements OnChanges, OnInit {
 
         // Automatic roation
         d3.timer( () => {
```

Comment:
Is there a way to remove the timer if `this.rotate === False`? Right now, this is still leaving the timer there, it just isn't doing anything. So, I'm wondering if we can bring the timer into and out of existence based on that flag.

Reply:
I think this gets directly at your play button comment above. Maybe the play button could just add/remove this timer? The play button could be something like the download icon and let angular do the subscription work.

Reply:
yeah, something like that. I was close (I just pushed the branch up, it's called play-button), but it needed more thought. 

I was also worried that I was mixing too many things: latest ap and rotation, so thought I'd do the minimum here and at least get the dates updating.